uvahelpr_exp/uvahelpr_exp/views.py

```diff
@@ -2,17 +2,17 @@
 import urllib.request
 import urllib.parse
 import json
+from django.http import JsonResponse
 
 def get_all_jobs(request):
     req = urllib.request.Request('http://models-api:8000/api/v1/jobs/')
     resp_json = urllib.request.urlopen(req).read().decode('utf-8')
     resp_dict = json.loads(resp_json)
-    filter = []
+    result = []
     for job in resp_dict['result']:
         newjob = {'id': job['id'], 'title': job['title'], 'description': job['description']}
-        filter.append(newjob)
-    result = json.dumps(filter)
-    return HttpResponse(result)
+        result.append(newjob)
+    return JsonResponse(result, safe=False)
```

Comment:
Ian said we actually want to keep this response as 'safe'. I guess we'll need to adjust the way we're processing the result as well in the layer up. So return this JSON as a dictionary (do like 'result' : array) and we should also have a status key in the response dict similar to the models api.